doc/wg/core/notes/core-notes-2023-01-13.md

```diff
@@ -0,0 +1,181 @@
+# Tock Core Notes 2023-01-13
+
+Attendees:
+- Leon Schuermann
+- Hudson Ayers
+- Chris Frantz
+- Philip Levis
+- Johnathan Van Why
+- Branden Ghena
+- Brad Campbell
+- Alexandru Radovici
+- Alyssa Haroldsen
+
+## Updates
+
+* None.
+
+## Open / Merged PRs
+
+* Hudson: Merged 3 minor PRs
+
+  - Fix Imix to not require process credential checking by default
+     ([#3376](https://github.com/tock/tock/pull/3376))
+  - Enable virtual function elimination by default for OpenTitan
+    ([#3358](https://github.com/tock/tock/pull/3358))
+  - [...] (TODO: PR number)
+
+  Opened PRs:
+
+  - `debug_writer_component`: make debug buffer size configurable; double
+    default debug buffer size ([#3369](https://github.com/tock/tock/pull/3369))
+  - `boards/qemu_rv32_virt`: set minimum reqd QEMU version to 7.2.0, fix
+    Makefile rules, add documentation
+    ([#3370](https://github.com/tock/tock/pull/3370))
+  - Implementation of PWM functionality for RP2040 ([#3372](https://github.com/tock/tock/pull/3372))
+  - Make the set_clocks functions of the RP2040 drivers public to crates only
+    ([#3373](https://github.com/tock/tock/pull/3373))
+
+* Branden: Why VFE for OpenTitan but not for others.
+
+* Hudson: Generally very useful, but still has soundness issues. OpenTitan
+  really cares about the code size, and the way they are using Tock seems to not
+  be affected by the soundness bugs.
+
+  One of the bugs is that, if you use it, but you don't rebuild the standard
+  library, that can lead to soundness issues. There was another issue where it
+  could be too eager in removing functions and removed some which were meant to
+  be called.
+
+* Johnathan: The people working on VFE are working together with the OpenTitan
+  team, so it's kind of using what they are building.
+
+## Approach to Propose Changes to Tock
+
+* Alyssa: What is the best format to propose changes to or ideas for Tock?
+  Presentation, Issue?
+
+* Johnathan: If it's more concrete a GH Issue might be a good place, otherwise
+  perhaps a presentation is better suited.
+
+* Phil: Lean towards presentation, mostly to put the emphasis on discussion.
+
+# License PR (TODO: PR number)
```

Comment:
I think this was supposed to be #3318

Reply:
Yikes, great catch, thanks. Also took care of the other missing PR numbers & links.